src/praisonai-agents/praisonaiagents/skills/manager.py

```
@@ -243,6 +243,315 @@ def load_resources(self, name: str) -> bool:
+            
+            # Write updated content
+            new_content = frontmatter + content
+            self._write_skill_atomically(skill_file, new_content)
+            
+            # Reload the skill
+            skill.instructions = None  # Clear cached content
+            self.activate(skill)
+            
+            return {"success": True, "skill": name}
+            
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)